deployment/README.md

```diff
@@ -1,160 +1,41 @@
 ## Deployment Instructions:
 
-- Additional dependencies:
-  - minimist - install at ./subgraphs at head of repository
-    - `npm install minimist`
-  - mustache
-    - `npm install mustache`
+# Deploys uniswap-v2 to ethereum in steegecs hosted service.
 
-```
-# Deploys uniswap-v2 from the uniswap-forks to mainnet in my hosted service.
-npm run deploy --SUBGRAPH=uniswap-forks --PROTOCOL=uniswap-v2 --NETWORK=mainnet --LOCATION=steegecs
+npm run deploy --PROTOCOL=uniswap-v2 --NETWORK=ethereum --TARGET=steegecs --SERVICE=h --TYPE=deploy
+npm run deploy --PROTOCOL=uniswap-v2 --NETWORK=ethereum --TARGET=steegecs --SERVICE=h
```

Comment:
omitting `--TYPE=deploy` will also deploy? If that is the case I could imagine that it could possible happen due to error. I think it would be better to make it explicit

Reply:
Okay - I can see how that would be more safe. Its a trade off for convenience, but I like can change it.